From 8dedbcdf4197eccd5e06c18a66a63d15615c8171 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Stefan=20Br=C3=BCns?= <stefan.bruens@rwth-aachen.de>
Date: Sun, 8 Apr 2018 22:24:21 +0200
Subject: [PATCH] Make automounting work even if StorageAccess is ignored

Summary:
There is no reason to make mounting dependent on the StorageAccess.ignore
flag, which just serves a a hint for visualization.

This change sneaked in when porting to KF5, no justification given.

BUG: 389479

Reviewers: ngraham, #plasma, broulik

Reviewed By: ngraham

Subscribers: plasma-devel

Tags: #plasma

Differential Revision: https://phabricator.kde.org/D12050
---
 solid-device-automounter/kded/DeviceAutomounter.cpp | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/solid-device-automounter/kded/DeviceAutomounter.cpp b/solid-device-automounter/kded/DeviceAutomounter.cpp
index 7a20bd5c..f1bc19d1 100644
--- a/solid-device-automounter/kded/DeviceAutomounter.cpp
+++ b/solid-device-automounter/kded/DeviceAutomounter.cpp
@@ -67,9 +67,6 @@ void DeviceAutomounter::automountDevice(Solid::Device &dev, AutomounterSettings:
 {
     if (dev.is<Solid::StorageVolume>() && dev.is<Solid::StorageAccess>()) {
         Solid::StorageAccess *sa = dev.as<Solid::StorageAccess>();
-        if (sa->isIgnored()) {
-            return;
-        }
 
         AutomounterSettings::setDeviceLastSeenMounted(dev.udi(), sa->isAccessible());
         AutomounterSettings::saveDevice(dev);
